src/app/manage/manage.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="mycards">
+    <div class="mycard">
+        <div class="card" class="{{user.isDeleted ? 'deleted-card' : 'active-card'}}" *ngFor="let user of users">
+            <span>{{user.firstName + " "+ user.lastName}}</span>
+            <button (click)="showDetails(user.id)" >Details</button>
+        </div>
+    </div>
+    <div>
```

Comment:
You should be using the same users-list component here. You can try passing an additional input to the list component so that it handles all the cases